### PR TITLE
[WIP] Precinct Z json 

### PR DESCRIPTION
**Pull request by Sehn-Knight**
Mod content only until z-levels are finished. It all works while testing it myself so let me know what you think of it.

### CODE REVIEW

**Comment by Malkeus** [comment]
Is this the RAW Precinct Z mod from the Mod List? If so, very cool! If not, very cool!

File: data/json/vehicles/carts.json [hunk 1]
```diff
@@ -1,6 +1,20 @@
 [
     {
         "type" : "vehicle",
+        "id" : "gurney",
+        "name" : "Gurney",
+		"blueprint" : "##",
```
Comment (illi-kun) [inline data/json/vehicles/carts.json]: Please remove tabs on this line and below by proper amount of spaces.

**Comment by mugling** [comment]
Stalled?

**Comment by Sehn-Knight** [comment]
Unfortunately, yes. I don't have any idea on how to go about balancing monsters and items in the structure. I tried to do something fairly large as my first addition to the game, so my bad.

**Pull request closed without merging**